docs/regressions-car17v1.5.md

```diff
@@ -40,19 +40,19 @@ nohup target/appassembler/bin/SearchCollection -index lucene-index.car17v1.5.pos
 
 nohup target/appassembler/bin/SearchCollection -index lucene-index.car17v1.5.pos+docvectors+rawdocs \
  -topicreader Car -topics src/main/resources/topics-and-qrels/topics.car17v1.5.benchmarkY1test.txt \
- -bm25 -axiom -rerankCutoff 20 -axiom.deterministic -output run.car17v1.5.bm25+ax.topics.car17v1.5.benchmarkY1test.txt &
+ -bm25 -rerankCutoff 20 -axiom -axiom.deterministic -output run.car17v1.5.bm25+ax.topics.car17v1.5.benchmarkY1test.txt &
```

Comment:
As a more general question, the parameter `-rerankCutoff` only applies when we specify `isRerank` to true. That means we should first see `args.rm3 || args.axiom || args.bm25prf` before seeing `-rerankCutoff`. So I guess we should put `rm3, axiom, bm25prf` before `-rerankCutoff` (the original way)?

Reply:
Should we do `rankCutoff` explicitly for each model? E.g., `rm3.rerankCutoff`?

Reply:
My original motivation was to keep all the "groups" together.

Reply:
Or I could make the `rankCutoff` parameter to the end?

Reply:
I don't think `rerankCutoff` for each model is necessary as they will probably not present at the same time? (what is the semantic for that?) 

Putting `rerankCutoff` at the end is fine